src/CalcViewModel/Common/KeyboardShortcutManager.cpp

```diff
@@ -474,8 +469,8 @@ const std::multimap<MyVirtualKey, WeakReference>& GetCurrentKeyDictionary(MyVirt
         }
         else
         {
-            auto iterViewMap = s_VirtualKeyControlInverseChordsForButtons.find(viewId);
-            if (iterViewMap != s_VirtualKeyControlInverseChordsForButtons.end())
+            auto iterViewMap = s_VirtualKeyInverseChordsForButtons.find(viewId);
```

Comment:
Good find! I wonder, does this change the behavior for shortcuts that we have documented?

Reply:
I tried all the shortcut using Ctrl+shift+Key, luckily, all of them had an equivalent Ctrl+Key